lib/ruby-rtf/parser.rb

```diff
@@ -283,7 +283,6 @@ def handle_control(name, val, src, current_pos)
       else
         unless @seen[name]
           @seen[name] = true
-          STDERR.puts "Unknown control #{name.inspect} with #{val} at #{current_pos}"
```

Comment:
Should we add some kind of logger or something? Or maybe add a debug flag and enable this in debug?